packages/slice-machine/lib/builders/SliceBuilder/index.tsx

```diff
@@ -62,14 +60,14 @@ const SliceBuilder: React.FunctionComponent = () => {
   };
 
   useEffect(() => {
-    if (Model.isTouched && isMounted) {
+    if (Model.isTouched) {
```

Comment:
was there a reason we wanted to check that the component was still mounted before changing the state?

Reply:
To be honest it's too old code, i don't know :x.

But
- the isMounted was a function so this was always true
- if we look at how the isMounted is done, calling a code in the use effect means the component is mounted already so it would always be true anyway.